kalite/facility/views.py

```diff
@@ -123,7 +123,7 @@ def _facility_user(request, facility, title, is_teacher=False, new_user=False, u
                 if request.next:
                     return HttpResponseRedirect(next)
                 else:
-                    zone_id = facility.get_zone().getattr(id, None)
+                    zone_id = facility.get_zone().id
```

Comment:
Can you change this to this:
```
getattr(facility.get_zone(), "id", None)
```

This should guarantee a result even if get_zone returns None.